client/layout/global-sidebar/header.tsx

```diff
@@ -30,7 +37,11 @@ export const GlobalSidebarHeader = () => {
 			) }
 			<span className="gap"></span>
 			<SidebarSearch
-				tooltip={ translate( 'Jump to…' ) }
+				tooltip={
+					hasEnTranslation( 'Search (%(shortcut)s)' )
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

Am I correct in assuming this conditional will be deleted once the translation is completed?

Reply:
Yes!